packages/create-vite-app/template/src/App.tsx

```diff
@@ -0,0 +1,41 @@
+import reactLogo from './assets/react.svg'
+import viteLogo from '/vite.svg'
+import './App.css'
+import {AutomergeUrl} from '@automerge/automerge-repo'
+import {useDocument} from '@automerge/automerge-repo-react-hooks'
+import { next as A } from '@automerge/automerge'
+
+interface CounterDoc {
+  counter: A.Counter
+}
+
+function App({docUrl}: {docUrl: AutomergeUrl}) {
+  const [doc, changeDoc] = useDocument<CounterDoc>(docUrl)
+
+  return (
+    <>
+      <div>
+        <a href="https://vitejs.dev" target="_blank">
+          <img src={viteLogo} className="logo" alt="Vite logo" />
+        </a>
+        <a href="https://react.dev" target="_blank">
+          <img src={reactLogo} className="logo react" alt="React logo" />
+        </a>
+      </div>
+      <h1>Vite + React</h1>
+      <div className="card">
+        <button onClick={() => changeDoc((d) => d.counter.increment(1))}>
+          count is { doc && doc.counter.value }
```

Comment:
I wonder if there's anything sufficiently small we could do that's more interesting/impressive than a counter...